File: src/C#/task_2/pangram.cs
using System;
using System.Collections.Generic;

class Program
{
    static bool IsPangram(string sentence)
    {
        // Создаем множество для хранения уникальных букв
        var uniqueLetters = new HashSet<char>();

        // Проходим по каждому символу в строке
        foreach (var c in sentence.ToLower()) // приводим буквы к нижнему регистру
        {
            // Проверяем, является ли символ буквой
            if (char.IsLetter(c))
            {
                // Добавляем букву в множество
                uniqueLetters.Add(c);
            }
        }

        // Панграмма должна содержать все буквы алфавита, то есть 26
        return uniqueLetters.Count == 26;
    }

    static void Main()
    {
        // Чтение строки из консоли
        Console.WriteLine("Введите строку:");
        string input = Console.ReadLine();

        // Вывод результата
        bool result = IsPangram(input);
        Console.WriteLine($"Output: {result}");
    }
}

File: src/Swift/task_2/pangram.swift
import Foundation

// Функция, которая проверяет, является ли строка панграммой
func isPangram(_ sentence: String) -> Bool {
    // Создаем множество для хранения уникальных символов
    let set = Set(sentence.lowercased().filter { $0.isLetter })

    // Проверяем, содержит ли множество 26 букв
    return set.count == 26
}

if let input = readLine() {
    print(isPangram(input))
}